integration/tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
+  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "target": "es2018",
-    "lib": ["es2018"],
-    "module": "commonjs",
-    "strict": true,
-    "outDir": "build",
-    "skipLibCheck": true,
-    "experimentalDecorators": true
+    "lib": ["es2018", "DOM"],
   },
-  "include": ["./"]
-}
+  "include": [
+    "**/*-test.ts"
```

Comment:
How about renaming this file to `tsconfig.tests.json` to highlight it's now only for the test files?

Reply:
Ah gotcha. Yeah, truly restructuring the generated code from the tests does sound like a good idea, i.e. its what I do in real projects anyway, but would be a large effort for probably just a small convenience/wrinkle at the moment, so I'm good with the current setup you've got. Thanks!